### DISCUSSION TITLE
How to transform an image to a specific artistic style?

### DISCUSSION THREAD

**Discussion by atishba** [lllyasviel/Fooocus#2112] [Q&A]
I want to take a picture like this :
<img width="651" alt="Screenshot 2024-01-31 at 10 10 34" src="https://github.com/lllyasviel/Fooocus/assets/101531753/a51f056a-9fea-4ded-aea0-ebdf759513ff">

And then transform it to a picture in a style like this without losing too much detail. What is the way to do it in fooocus? Or would Photoshop be better in my case?

<img width="533" alt="Screenshot 2024-01-15 at 20 54 58" src="https://github.com/lllyasviel/Fooocus/assets/101531753/66fe4daa-5e4f-4c7f-a653-68635be6de4f">

**Comment by mashb1t** [comment]
you can use CDPS / PyraCanny in the image prompt advanced section for that + a style and a prompt

**Reply by iomarcovalente** [reply]
that's exactly it. I have tried and it worked very well, as long as the prompt is consistent with the original image

**Comment by robinwang25** [comment]
I am also very curious.
